src/flb_engine.c

```diff
@@ -719,6 +720,29 @@ int flb_engine_start(struct flb_config *config)
     }
     config->evl_bktq = evl_bktq;
 
+    /*
+     * Event loop channel to send context signals.
+     *
+     */
+    /* Create the event loop and set it in the global configuration */
+    config->ctx_evl = mk_event_loop_create(8);
```

Comment:
Please change this so the event loop size is a constant that can be changed in a central place like the rest of them.